packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -26,7 +26,21 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
     const { className, styles, iconName, imageErrorAs, theme } = this.props;
     const isPlaceholder = typeof iconName === 'string' && iconName.length === 0;
     const isImage = this.props.iconType === IconType.image || this.props.iconType === IconType.Image || !!this.props.imageProps;
-    const { iconClassName, children } = getIconContent(iconName);
+    let iconContent = undefined;
+
+    if (this.props.doNotCacheNullIcon) {
```

Comment:
You can use tertiary operator for single line assignment and declaration: let iconContent = this.props.don. ? <> : <>. 
You can also keep the variable assignment as: const { iconClassName, children } = iconContent || {}

Reply:
Done